tests/test_release_process.py

```diff
@@ -853,16 +853,34 @@ def test_bump_minor_release_from_pending_patch_release_pr(self,
     def test_get_log_string(self):
         version_pieces = ['0', '10', '2']
         self.assertEqual('0.10.2...0.10.1',
-                         release_process._get_log_string(version_pieces))
+                         release_process._get_log_string(version_pieces,
+                                                         "0.10.2"))
         version_pieces = ['0', '3', '0']
         self.assertEqual('0.3.0...0.2.0',
-                         release_process._get_log_string(version_pieces))
+                         release_process._get_log_string(version_pieces,
+                                                         "0.3.0"))
         version_pieces = ['0', '3', '25']
         self.assertEqual('0.3.25...0.3.24',
-                         release_process._get_log_string(version_pieces))
+                         release_process._get_log_string(version_pieces,
+                                                         "0.3.25"))
         version_pieces = ['0', '25', '0']
         self.assertEqual('0.25.0...0.24.0',
-                         release_process._get_log_string(version_pieces))
+                         release_process._get_log_string(version_pieces,
+                                                         "0.25.0"))
+
+    def test_get_log_string_prerelease(self):
+        version_pieces = ['0', '25', '0']
+        self.assertEqual('0.25.0rc1...0.24.0',
+                         release_process._get_log_string(version_pieces,
+                                                         "0.25.0rc1"))
+        version_pieces = ['0', '25', '0']
+        self.assertEqual('0.25.0rc2...0.24.0',
+                         release_process._get_log_string(version_pieces,
+                                                         "0.25.0rc2"))
+        version_pieces = ['0', '25', '0']
+        self.assertEqual('0.25.0b1...0.24.0',
+                         release_process._get_log_string(version_pieces,
+                                                         "0.25.0b1"))
```

Comment:
Do we definitely want `0.24.0` to be the "version before" `0.25.0rc2`, or should it be `0.25.0rc1` for release-note generation?

Reply:
I wasn't sure exactly how we wanted to handle this. This is just for the changelog in the release page like: https://github.com/Qiskit/qiskit-terra/releases/tag/0.20.0

For rc1 and the full release I'm pretty sure we want the full thing but rc2 I could see the value in making it strictly incremental over rc1. I don't really have a strong opinion as I never really understood why people like the changelog view over the full release notes (but some people do use it apparently)

Reply:
I think for a changelog, `rc2` should only be changes above `rc1`, probably.  The full release notes will always have everything, and I guess the type of person looking at the changelog for a second release candidate probably knows what was in the first release candidate already.